src/Datadog.Trace/Datadog.Trace.csproj

```diff
@@ -78,5 +78,8 @@
     <PackageReference Include="System.Reflection.Emit" Version="4.3.0" />
     <PackageReference Include="System.Reflection.Emit.Lightweight" Version="4.3.0" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
I think you can remove this change and the project will still compile correctly

Reply:
Agree, removed.